installerscript/appdaemon.yaml

```diff
@@ -0,0 +1,41 @@
+logs:
+  main_log:
+    filename: /home/appdaemon/.appdaemon/log/appdaemon.log
+  access_log:
+    filename: /home/appdaemon/.appdaemon/log/access.log
+  error_log:
+    filename: /home/appdaemon/.appdaemon/log/error.log
+  diag_log:
+    filename: /home/appdaemon/.appdaemon/log/diag.log
+    log_generations: 5
+    log_size: 1024
+    format: "{asctime} {levelname:<8} {appname:<10}: {message}"
+appdaemon:
+  time_zone: <your timezone>    ### Example Europe/Oslo
+  latitude: <your latitude in form of xx.xxxxxx>
+  longitude: <your longitude in form of xx.xxxxxx>
+  elevation: <your elevation in meters>
+  plugins:
+    HASS:
+      type: hass
+      ha_url: http:// (or https) <ip address of HA>:8123 (or your custom port)
+      token: <your long-lived access token created in HA>
+      ### You must create a long-lived token in HA for AppDaemon to be used here
+
+    MQTT: 
+      type: mqtt
+      namespace: mqtt
+      client_host: <ip address to your MQTT broker>
+      client_user: <username to your MQTT broker>
+      client_password: <password to your MQTT broker>
+      client_topics:
```

Comment:
We can leave out the client topics, as AD will subscribe to all topics on the broker by default

Reply:
Fixed